source/extensions/omni.isaac.lab/omni/isaac/lab/assets/rigid_object/rigid_object_data.py

```diff
@@ -16,9 +16,17 @@ class RigidObjectData:
     """Data container for a rigid object.
 
     This class contains the data for a rigid object in the simulation. The data includes the state of
-    the root rigid body and the state of all the bodies in the object. The data is stored in the simulation
+    the root rigid body and the state of all the bodies in the object.The data is stored in the simulation
     world frame unless otherwise specified.
 
+    For a rigid body, there are two frames of reference that are used:
+
+    - Actor frame: The frame of reference of the actor that contains the rigid body.
```

Comment:
Shall we mention that this coincides with the Xform on which the rigid body is attached? 
I guess people do not necessarily know the concept of actors.

Reply:
I found xform frame in the docs to be a bit confusion. Left it as actor frame for now.